### DISCUSSION TITLE
API for testing etc.

### DISCUSSION THREAD

**Discussion by Jerac102** [dotnet/ResXResourceManager#674] [General]
Hi,
  i would like to know if there is some API implemented? What i want to do is to, e.g. to test my Excel file format if it can be imported without any problems into ResX Manager. Also other automation would be nice. Now i have to do it manually, what is boring and takes time wheres running (unit) test are do it in milliseconds.

Best regards!

**Comment by tom-englert** [comment]
https://github.com/dotnet/ResXResourceManager/blob/master/Documentation%2FTopics%2FScripting.md
